rfc/github-org-migration.md

```diff
@@ -0,0 +1,755 @@
+    - It SHOULD NOT be archived.
+    - It SHOULD be migrated.
+- [ ] [w3up](https://github.com/web3-storage/w3up)
+    - Implementation of the protocols and specs repos. The current web3.storage system.
+    - It SHOULD NOT be archived.
+    - It SHOULD be migrated.
+- [ ] [ucanto-name-system](https://github.com/web3-storage/ucanto-name-system)
+    - [ ] check with @gobengo what should be done
+- [ ] [alt-gateway](https://github.com/web3-storage/alt-gateway)
+    - It SHOULD NOT be archived.
+    - It SHOULD NOT be migrated.
+- [ ] [link.web3.storage](https://github.com/web3-storage/link.web3.storage)
+    - [ ] check with @vascosantos10 if it can be archived
+- [ ] [dag-check](https://github.com/web3-storage/dag-check)
+    - It SHOULD be archived.
+    - It SHOULD NOT be migrated.
+- [ ] [pickup](https://github.com/web3-storage/pickup)
+    - Used by nft.storage for the pinning service API (only supported by them)
+    - It SHOULD NOT be archived.
+    - It SHOULD NOT be migrated.
+- [ ] [workers](https://github.com/web3-storage/workers)
+    - [ ] check with @vascosantos10 if it can be archived
+- [ ] [minibus](https://github.com/web3-storage/minibus)
+    - There's a deployed service with this but we're not using it in production. A small data service and it works with blocks for CIDs. Works for smaller things and not for big DAGs
+    - It SHOULD NOT be archived.
+    - It SHOULD NOT be migrated.
+- [ ] [w3name](https://github.com/web3-storage/w3name)
+    - Used in production for IPNS.
+    - It SHOULD NOT be archived.
+    - It SHOULD NOT be migrated.
+- [ ] [reads](https://github.com/web3-storage/reads)
+    - Frontend for the racing GW system. Also includes a denylist that gets checked before racing. A superset of badbits.
+    - It SHOULD NOT be archived.
+    - It SHOULD NOT be migrated.
+- [ ] [claudio](https://github.com/web3-storage/claudio)
+    - Original MVP for hoverboard.
+    - It SHOULD be archived.
+    - It SHOULD NOT be migrated.
+- [ ] [minibus-client](https://github.com/web3-storage/minibus-client)
+    - The cli for minibus.
+    - It SHOULD NOT be archived.
+    - It SHOULD NOT be migrated.
+- [ ] [w3link](https://github.com/web3-storage/w3link)
+    - [ ] check with @vascosantos10 if it can be archived
+- [ ] [dagula](https://github.com/web3-storage/dagula)
+    - Used by freeway and hoverboard. lib and cli for fetching a DAG from a remote peer.
+    - It SHOULD NOT be archived.
+    - It SHOULD NOT be migrated.
+- [ ] [dagula-gateway](https://github.com/web3-storage/dagula-gateway)
+    - Experiment to back a GW with Elastic IPFS over Bitswap, but it didn't work out with the amount of traffic.
+    - It SHOULD be archived.
+    - It SHOULD NOT be migrated.
+- [x] [fast-unixfs-exporter](https://github.com/web3-storage/fast-unixfs-exporter)
+    - already archived.
+- [ ] [telemetry-grafana-agent](https://github.com/web3-storage/telemetry-grafana-agent)
+    - [ ] check with @vascosantos10 if it can be archived
```

Comment:
no, this holds a bunch of our Grafana metrics, including w3up, gateways, cloudflare metrics, etc

Reply:
note this is deployed in a digital ocean app